parallel-install/pkg/preinstaller/mocks/resourceparsermock.go

```diff
@@ -0,0 +1,92 @@
+package mocks
```

Comment:
For autogenerated and more flexible mocks, please check `mockery`.
You can find a full usage here:
https://github.com/kyma-project/cli/blob/bad193c82b5fa538f69335789e64bb536beb2200/internal/kube/kube.go#L15-L16

and how to use it:
https://github.com/kyma-project/cli/blob/fa37f116219247daae3487abad5499cc6fa939ec/cmd/kyma/create/system/system_test.go#L67-L68 

Writing your own mocks is a lot of unnecessary work and it adds more maintenance effort as the code evolves.
Generated mocks take all that away.